src/main/java/org/selenium/pages/WishListPage.java

```diff
@@ -0,0 +1,82 @@
+package org.selenium.pages;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.util.Locale;
+import java.util.Random;
+
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
+
+
+public class WishListPage extends BasePage {
+    private Random random = new Random();
+    private Alert alert;
+    private By table = By.xpath("//div[@id='block-history']");
+    private By name = By.xpath("//tr/td[1]/a");
+    private By buttonDelete = By.xpath("//a[@class='icon']");
+    private By topSellersList = By.xpath("//ul[@class='block_content products-block']");
+    private By nameList = By.xpath("//input[@id='name']");
+    private By saveButton = By.xpath("//button[@id='submitWishlist']");
+    private By productContainer = By.xpath("//ul[@class='row wlp_bought_list']");
+    private By productName = By.xpath("//a[@title='Product detail']");
+
+
+    public WishListPage createListAutomatically() {
+        if (isElementPresent(table) == true) {
+            driver.findElement(buttonDelete).click();
+            alert = wait.until(ExpectedConditions.alertIsPresent());
+            alert.accept();
+        }
+        choosingProduct();
+        return this;
```

Comment:
1) Why when choosingProduct() returns ProductDetailPage , you return WishListPage  page
2) This method doesn't create wishlist